go/worker/storage/committee/policy.go

```diff
@@ -10,29 +11,38 @@ import (
 var (
 	computeCommitteePolicy = &committee.AccessPolicy{
 		Actions: []accessctl.Action{
-			"Apply",
-			"ApplyBatch",
+			accessctl.Action(api.MethodApply.Full()),
+			accessctl.Action(api.MethodApplyBatch.Full()),
 		},
 	}
 	txnSchedulerCommitteePolicy = &committee.AccessPolicy{
 		Actions: []accessctl.Action{
-			"Apply",
-			"ApplyBatch",
+			accessctl.Action(api.MethodApply.Full()),
+			accessctl.Action(api.MethodApplyBatch.Full()),
 		},
 	}
 	mergeCommitteePolicy = &committee.AccessPolicy{
 		Actions: []accessctl.Action{
-			"Merge",
-			"MergeBatch",
+			accessctl.Action(api.MethodMerge.Full()),
+			accessctl.Action(api.MethodMergeBatch.Full()),
 		},
 	}
 	// NOTE: GetDiff/GetCheckpoint need to be accessible to all storage nodes,
 	// not just the ones in the current storage committee so that new nodes can
 	// sync-up.
 	storageNodesPolicy = &committee.AccessPolicy{
 		Actions: []accessctl.Action{
-			"GetDiff",
-			"GetCheckpoint",
+			accessctl.Action(api.MethodGetDiff.Full()),
+			accessctl.Action(api.MethodGetCheckpoint.Full()),
+		},
+	}
+	sentryNodesPolicy = &committee.AccessPolicy{
```

Comment:
Is there a case where sentry nodes would not be allowed to proxy all methods?

Reply:
no, but this is exactly what this specifies (these are all the methods that use access policies)